app/src/main/res/drawable/ic_launcher_dev_background.xml

```diff
@@ -0,0 +1,84 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please move this only to `dev` flavor from `main`

Reply:
done